src/views/Support.vue

```diff
@@ -219,5 +225,90 @@ export default {
 			}
 		},
 	},
+	computed: {
+		getInstalledComponents() {
+			let componentText = "";
+			const devices = this.getWildcardTopics(
+				"openWB/system/device/+/config",
+			);
+			for (const [, deviceConfig] of Object.entries(devices)) {
+				const deviceId = deviceConfig.id;
+				const deviceName = deviceConfig.name;
+				componentText += `DID: ${deviceId}, ${deviceName}\n`;
+
+				const components = this.getWildcardTopics(
+					`openWB/system/device/${deviceId}/component/+/config`,
+				);
+
+				for (const [, componentConfig] of Object.entries(components)) {
+					const componentId = componentConfig.id;
+					const componentName = componentConfig.name;
+					const componentType = componentConfig.type;
+					const manufacturer =
+						componentConfig.info?.manufacturer || "N/A";
+					const model = componentConfig.info?.model || "N/A";
+					componentText += `  CID: ${componentId}, ${componentName}, Type: ${componentType}, Manufacturer: ${manufacturer}, Model: ${model}\n`;
+				}
+				componentText += "\n"; // Add an extra newline between devices
+			}
+			return componentText.trim();
+		},
+		getSerialNumber() {
+			const serial = this.getWildcardTopics(
+				"openWB/system/serial_number",
+			);
+			const serialKey = Object.keys(serial);
+			return serial[serialKey];
+		},
+		getVehicleDetails() {
+			let vehicleText = "";
+			const vehicles = this.getWildcardTopics("openWB/vehicle/+/name");
+			console.log(vehicles);
+			for (const [vehicleTopic, vehicleValue] of Object.entries(
+				vehicles,
+			)) {
+				const vehicleId = vehicleTopic.split("/")[2];
+				const vehicleName = vehicleValue;
+				console.log(vehicleId);
+				console.log(vehicleValue);
+				vehicleText += `ID: ${vehicleId} Name:${vehicleName}\n`;
```

Comment:
Die Formatierung mit Zeilenumbruch und später in Z. 286 Einrückung sieht im Textfeld nett aus, geht aber im Systembericht leider verloren. Daher besser pro Fahrzeug alle Daten in einer Zeile erstellen.

Um es übersichtlicher zu machen, kann die komplette Zeile zu einem Fahrzeug in einer Operation zusammengefasst werden. Also die Zeilen 274 und 286 entfernen und in Zeile 288 den kompletten Text erzeugen.
```javascript
vehicleText += `ID: ${vehicleId} Name: ${vehicleName} Man.: ${make} Mod.: ${model}\n`
```